utils/utils.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+// this is code taken from the v2 branch of github.com/deis/deis/tests/utils
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+)
+
+// RunCommandWithStdoutStderr execs a command and returns its output.
+func RunCommandWithStdoutStderr(cmd *exec.Cmd) (bytes.Buffer, bytes.Buffer, error) {
+	var stdout, stderr bytes.Buffer
+	stderrPipe, err := cmd.StderrPipe()
+	stdoutPipe, err := cmd.StdoutPipe()
+
+	cmd.Env = os.Environ()
+	if err != nil {
+		fmt.Println("error at io pipes")
+	}
+
+	err = cmd.Start()
+	if err != nil {
+		fmt.Println("error at command start")
+	}
+
+	go func() {
+		streamOutput(stdoutPipe, &stdout, os.Stdout)
+	}()
+	go func() {
+		streamOutput(stderrPipe, &stderr, os.Stderr)
+	}()
+	err = cmd.Wait()
+	if err != nil {
+		fmt.Println("error at command wait")
+	}
+	return stdout, stderr, err
+}
+
+// streamOutput from a source to a destination buffer while also printing
```

Comment:
@bacongobbler you're right. all code in this file was copied directly from https://github.com/deis/deis/blob/v2/tests/utils/utils.go, so that upstream projects don't have to depend on `github.com/deis/deis` to get it (which pulls in a lot of unnecessary dependencies).

This is a good opportunity to improve - commits coming